graf2d/gpadv7/v7/inc/ROOT/RCanvas.hxx

```diff
@@ -122,16 +124,19 @@ public:
    const std::string &GetTitle() const { return fTitle; }
 
    /// Set the canvas's title.
-   RCanvas &SetTitle(const std::string &title) {
+   RCanvas &SetTitle(const std::string &title)
+   {
       fTitle = title;
       return *this;
    }
 
+   void ResolveSharedPtrs();
+
    /// Convert a `Pixel` position to Canvas-normalized positions.
-   std::array<RPadLength::Normal, 2> PixelsToNormal(const std::array<RPadLength::Pixel, 2> &pos) const final
-   {
-      return {{pos[0] / fSize[0], pos[1] / fSize[1]}};
-   }
+//   std::array<RPadLength::Normal, 2> PixelsToNormal(const std::array<RPadLength::Pixel, 2> &pos) const final
```

Comment:
So - how else? :-)

Reply:
Sorry, I comment out code when trying different implementations of RPadLength. Now it is re-enabled again with proper calls. One should revise it while user coordinates may not be present in RPadLength - also pixel coordinates can be disabled.